src/EnergyPlus/SetPointManager.cc

```diff
@@ -3917,6 +3921,16 @@ namespace SetPointManager {
 						// Call the CALC routine, with an optional argument to only set
 						// the initialization NODE(:)% setpoint, and not the OutAirSetPtMgr(:)%SetPt
 						OutAirSetPtMgr( SetPtMgrNum ).calculate( NodeNum, true );
+						Node( NodeNum ).TempSetPointLo = OutAirSetPtMgr( SetPtMgrNum ).SetPtLo;
```

Comment:
Right.  Then perhaps the function should be changed to simply return the value.  e.g.
```
Node( NodeNum ).TempSetPoint = OutAirSetPtMgr(SetPtMgrNum).calculate;
```